core/src/main/java/hudson/FileSystemProvisioner.java

```diff
@@ -46,8 +46,6 @@
 /**
  * Prepares and provisions workspaces for {@link AbstractProject}s.
  *
- * <p>
- *
```

Comment:
Please keep the diff minimal.

Reply:
Some javadoc changes are needed otherwise the build fail.